justfile

```diff
@@ -4,7 +4,7 @@ alias l := lint-all
 
 # Adding a chain
 add-chain:
-	go run ./add-chain
+	go run ./add-chain add-new-chain
```

Comment:
Wondering if it would make sense to rename the top level cmd to something more general like `scr-ops`? Don't feel strongly, though.

Reply:
I was also thinking this, but didn't feel confident making a larger change like that. do you think we should do that as part of this PR, or as follow-up work?

Reply:
i've gone ahead and made this change in #725

Reply:
Thanks. I think in a follow-up makes sense. We should also get @bitwiseguy's opinion.